src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs

```diff
@@ -308,6 +307,24 @@ private static string ExportXRefMap(DocumentBuildParameters parameters, Document
             return xrefMapFileNameWithVersion;
         }
 
+        private static string GetXrefMapFileNameWithVersion(string version)
+        {
+            if (string.IsNullOrEmpty(version))
+            {
+                return XRefMapFileName;
+            }
+
+            // TODO: full check of invalid chars
+            version = Regex.Replace(
```

Comment:
In monikerrange, write to `> netcore1.0.xrefmap.yml` will fail... Exception is thown by `Path.Combine` before passed into serializer.